PerformanceTests/JetStream2/in-depth.html

```diff
@@ -113,12 +113,20 @@ <h2>In-Depth Analysis</h2>
         </p>
 
         <p>
-        Note that scores from JetStream 2 are not comparable to scores to other versions
+        JetStream 2.1 runs the same benchmarks as JetStream 2, but updates the benchmark driver to
+        improve score stability. This is achieved by pre-fetching network resources to warm up the
+        browser network cache prior to running the benchmarks. This reduces perturbations on the
+        measurement of JavaScript execution time due to network latency and any consequent second
+        order effects.
```

Comment:
maybe instead of "this reduces" we could say "This can reduce" or "This may reduce"

Reply:
Fixed.